lib/shared/permissions.js

```diff
@@ -145,22 +146,13 @@ exports.elevateCommand = (command, options) => {
   const prefixedCommand = _.concat(exports.getEnvironmentCommandPrefix(options.environment), command);
 
   if (os.platform() === 'win32') {
-    const elevator = Bluebird.promisifyAll(require('elevator'));
-
-    return elevator.executeAsync(prefixedCommand, {
-      hidden: true,
-      terminating: true,
-      doNotPushdCurrentDirectory: true,
-      waitForTermination: true
-    }).then(() => {
+    const elevator = Bluebird.promisifyAll(bindings.load('elevator'));
```

Comment:
The thing is that the location of the final add-on depends on the platform, and on the gyp version, so you would have to implement some non-trivial logic to validate this.